pkg/minishift/openshift/service.go

```diff
@@ -64,41 +104,45 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 	cmdName := instanceState.InstanceConfig.OcPath
 	cmdOut, err := runner.Output(cmdName, tokens...)
 	if err != nil {
-		return "", errors.New(fmt.Sprintf("No service with name '%s' defined in namespace '%s'", service, namespace))
+		return "", errors.New(fmt.Sprintf("Service with name '%s' in namespace '%s' doesn't have route associated", service, namespace))
```

Comment:
I don't think this is going to work. Now it displays the right thing for say the router service, but I also get 'Service with name 'foo' in namespace 'default' doesn't have route associated' for some random non existent service. Really we need to differentiate somewhere.